codegen/lib/graphql_swift_gen.rb

```diff
@@ -221,9 +221,45 @@ def deserialize_value_code(field_name, expr, type, untyped: true)
       raise NotImplementedError, "Unexpected #{type.kind} argument type"
     end
   end
-
+  
+  def generate_input_init(type) 
+    text = "public init("
+      input_fields = type.required_input_fields + type.optional_input_fields
+      input_fields.each do |field|
+        text << escape_reserved_word(field.camelize_name)
+        text << ": "
+        text << swift_input_type(field.type)
+        text << (field.type.non_null? ? "" : " = nil")
+        text << (field == input_fields.last ? "" : ", ")
+      end
+    text << ")"
+    text << " {\n"
+      type.input_fields.each do |field|
+        name = escape_reserved_word(field.camelize_name)
+        text << "self." + name + " = " + name
+        text << "\n"
+      end
+    text << "}"
+  end
```

Comment:
It looks like you just converted the code from the ERB file into a ruby method with string concatenation.  I find the code more readable the way it was before.  I also find PRs with miscellaneous changes like this a lot harder to review, since a refactor can hide functional changes.